WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -390,8 +390,6 @@ d_sky_lords = {
 	# Parent Religion 
```

Comment:
This file is not ready for implementation for a series of reasons:

1. These changes need a detailed explanation in a dedicated issue with explanations on why this was needed and what the aim with the changes were. 
2. Some of the changes need some tinkering on balance.
3. There has been a lot of reshuffling, which we do in separate files to avoid compatibility problems.